_posts/2019-11-20-bazel-1.2.0.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: posts
+title: "Bazel 1.2"
+authors:
+  - philwo
+---
+
+We've just released [Bazel 1.2](https://github.com/bazelbuild/bazel/releases/tag/1.1.0)!
```

Comment:
I guess Bazel 1.2 should link to release tag 1.2.0 :)

Reply:
Good catch ;) Thanks!